src/main/resources/assets/minestuck/models/item/temple_scanner.json

```diff
@@ -1,6 +1,19 @@
 {
-    "parent": "item/generated",
-    "textures": {
-        "layer0": "minestuck:item/temple_scanner"
-    }
+  	"parent": "minecraft:item/generated",
+  	"textures": {
+    	"layer0": "minestuck:item/temple_scanner/temple_scanner_base"
+  	},
+  	"overrides": [
+  		{ "predicate": { "damage": 0 }, "model": "minestuck:item/temple_scanner_base" },
+  		{ "predicate": { "damage": 0.0001, "angle": 0.0000 }, "model": "minestuck:item/temple_scanner_north" },
+        { "predicate": { "damage": 0.0001, "angle": 0.0625 }, "model": "minestuck:item/temple_scanner_northeast" },
+        { "predicate": { "damage": 0.0001, "angle": 0.1875 }, "model": "minestuck:item/temple_scanner_east" },
+        { "predicate": { "damage": 0.0001, "angle": 0.3125 }, "model": "minestuck:item/temple_scanner_southeast" },
+        { "predicate": { "damage": 0.0001, "angle": 0.4375 }, "model": "minestuck:item/temple_scanner_south" },
+        { "predicate": { "damage": 0.0001, "angle": 0.5625 }, "model": "minestuck:item/temple_scanner_southwest" },
+        { "predicate": { "damage": 0.0001, "angle": 0.6875 }, "model": "minestuck:item/temple_scanner_west" },
+        { "predicate": { "damage": 0.0001, "angle": 0.8125 }, "model": "minestuck:item/temple_scanner_northwest" },
+        { "predicate": { "damage": 0.0001, "angle": 0.9375 }, "model": "minestuck:item/temple_scanner_north" },
```

Comment:
This may just be how it needs to be, but is there a reason 0.0001 was picked for the damage?

Reply:
Yeah; if damage is 0, we'd want the scanner to show the "turned off" sprite. In order to do this with predicates, the best I can do is to set a value close to zero as the first value to get checked.